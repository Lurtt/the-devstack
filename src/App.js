import React, { Component } from 'react'

import 'styles/normalize'
import { Example } from 'components'

class App extends Component {
  render() {
    return <Example />
  }
}

export default App
