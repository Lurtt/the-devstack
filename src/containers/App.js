import React, { Component } from 'react'

import 'styles/normalize'
import { Layout } from 'components'

class App extends Component {
  render() {
    return (
      <Layout>
        <div>here is the content</div>
      </Layout>
    )
  }
}

export default App
