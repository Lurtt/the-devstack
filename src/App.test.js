import App from './App'

jest.mock('./components/Loading', () => 'Loading')

describe('<App />', () => {
  test(`Should has 'from' field `, () => {
    const app = new App()

    expect(app.from).toEqual(0)
  })
})
