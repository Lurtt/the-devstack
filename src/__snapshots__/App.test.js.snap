// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should create instance 1`] = `
<h1>
  hi there!
</h1>
`;
